.travis.yml

```diff
@@ -177,6 +168,9 @@ after_success:
 
 notifications:
   irc:
+    if: repo = haskell/cabal
```

Comment:
are this conditionals documented somewhere, some recent Travis functionality? (single equals is confusing!)

Reply:
Yep, https://docs.travis-ci.com/user/notifications#conditional-notifications mentions them in the context of notifications and https://docs.travis-ci.com/user/conditions-v1 goes more in detail on them.

Reply:
Nice, and "oh dear, soon .travis.yml will be turing complete (if it's not already)"

Reply:
It would be nice to be able to say

```yaml
script:
  - if: os = linux
    then: "bash bla bla"
  - do this on osx and linux still
```

But I guess we cannot. (it's not so useful for `Cabal`, but elsewhere).

Reply:
I think you should be able to do that with the `TRAVIS_OS_NAME` env var.  So something like

```yaml
script:
  - [ "$TRAVIS_OS_NAME" = "linux" ] && bla bla
  - do this on osx and linux still
```

Reply:
Yes, that what I do now; it gets tedious :)